.github/workflows/build.yml

```diff
@@ -17,37 +18,32 @@ jobs:
       - name: Create a virtual environment for Zephyr and intall west
         run: |   
           python3 -m venv $GITHUB_WORKSPACE/zephyrproject/.venv
-          source $GITHUB_WORKSPACE/zephyrproject/.venv/bin/activate
-          pip install west
+          $GITHUB_WORKSPACE/zephyrproject/.venv/bin/pip install west
 
       - name: Get Zephyr source code
         run: |
-          source $GITHUB_WORKSPACE/zephyrproject/.venv/bin/activate
-          west init -m https://github.com/eurus-project/zephyr $GITHUB_WORKSPACE/zephyrproject/
+          $GITHUB_WORKSPACE/zephyrproject/.venv/bin/west init -m https://github.com/eurus-project/zephyr \
```

Comment:
This is less error-prone but I think it's a bit less readable. Still, we can leave it.